Source/Scene/Cesium3DTileset.js

```diff
@@ -1311,50 +1372,47 @@ define([
                     }
 
                     if (!allChildrenLoaded) {
-                        // Tile does not meet SSE.  Add its commands since it is the best we have and request its children.
-                        selectTile(tileset, t, fullyVisible, frameState);
+                        childrenVisibility = computeChildrenVisibility(t, frameState, false);
```

Comment:
Why isn't `useChildrenBoundUnion` called before `computeChildrenVisibility` to save the CPU overhead when it isn't needed?